tests/devices.py

```diff
@@ -288,3 +288,22 @@
         }
     ],
 }
+
+SIGNAL_REPEATER = {
+    "15014": [{"9003": 0}],
+    "3": {
+        "0": "IKEA of Sweden",
+        "1": "TRADFRI signal repeater",
+        "2": "",
+        "3": "2.0.019",
+        "6": 1,
+    },
+    "5750": 6,
+    "9001": "Signalforstaerker",
+    "9002": 1566141249,
+    "9003": 65600,
+    "9019": 1,
+    "9020": 1566401800,
+    "9054": 0,
+    "9084": " 83 6f b7 c 7a f4 8a 14 4a 94 4a 94 41 e0 a2 4f",
```

Comment:
Do you know what 9084 is?

Reply:
Unfortunately not. I've been trying to research it to no avail. A private key, or something to do with Homekit or Alexa maybe?

Reply:
Just found this resource but it's not there either:
https://github.com/glenndehaan/ikea-tradfri-coap-docs